runtime/src/snapshot_utils.rs

```diff
@@ -239,10 +239,11 @@ pub fn remove_tmp_snapshot_archives(snapshot_archives_dir: &Path) {
     }
 }
 
-/// Make a full snapshot archive out of the AccountsPackage
-pub fn archive_snapshot_package(
+/// Make a snapshot archive out of the AccountsPackage
+pub(crate) fn archive_snapshot_package(
```

Comment:
A nice fallout was that I can reduce the visibility into some `snapshot_utils` functions, since this functionality is contained within the traits in the previous comment.